addons/mail/static/src/components/chatter_topbar/chatter_topbar.xml

```diff
@@ -14,6 +14,8 @@
                             }"
                             t-att-disabled="chatter.isDisabled"
                             t-on-click="_onClickSendMessage"
+                            title="Send a message"
+                            data-hotkey="m"
```

Comment:
For all template changes in this PR, please check new coding style rules.

Reply:
The new guideline is still under debate, so I would prefer if this followed the guideline that is applied on that file right now, which would be to have `title` and `data-hotkey` after `t-att-disable` but before `t-on-click`.

We'll do a pass on all our files to apply the new guideline later.